CannibalisticZombies/Assets/PlayerCharacterController.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCharacterController : MonoBehaviour
+{
+    public float moveSpeed;
+
+    public Transform orientation;
+
+    float horizontalInput;
+    float verticalInput;
+
+    Vector3 moveDir;
+
+    Rigidbody rb;
+
```

Comment:
For **every** class and method, we want to put a comment block at the top. We can use it to define the method, but even if we don't put any information it makes the code much easier to read.

Ex:
```
///-/////////////////////////////////////////////////////////////////////
///
/// This class defines the class blah blah blah
/// 
public class MyClass {
   ///-/////////////////////////////////////////////////////////////////////
   ///
   public void Start {
   
   }
}
```